src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -22,8 +22,8 @@ private void TestWindowApisUnix()
             // This tests the hardcoded implementations of these APIs on Unix.
             using (Process p = Process.GetCurrentProcess())
             {
-                Assert.True(p.Responding);
-                Assert.Equal(string.Empty, p.MainWindowTitle);
+                Assert.Throws<PlatformNotSupportedException>(() => p.Responding);
```

Comment:
Yes please - as discussed offline, we have to assume code is not willing to treat platforms differently (nor should it). So if calling code can reasonably continue if we return some "default" value -- appears to be true in this case -- we should do that. If calling code cannot likely continue successfully -- for example, we return an object it is likely to try to use, and that won't work -- we should throw PNSE so it fails in a clear and prompt manner.